shared/contract-utils/contracts/interfaces/IERC20WithCheckpointing.sol

```diff
@@ -8,6 +8,6 @@ import "@aragon/os/contracts/lib/token/ERC20.sol";
 
 
 contract IERC20WithCheckpointing is ERC20 {
-    function balanceOfAt(address _owner, uint256 _blockNumber) public view returns (uint256);
-    function totalSupplyAt(uint256 _blockNumber) public view returns (uint256);
+    function balanceOfAt(address _owner, uint64 _blockNumber) public view returns (uint256);
```

Comment:
Given that MiniMe uses `uint256` as its time value in public interfaces, we shouldn't change this interface to `uint64`.